packages/cli/src/utils/convert.js

```diff
@@ -97,35 +97,46 @@ const getAuthFieldKeys = appDefinition => {
   return fieldKeys;
 };
 
-const renderLegacyPackageJson = async (legacyApp, appDefinition) => {
+const renderPackageJson = async (appInfo, appDefinition) => {
+  const name = _.kebabCase(
+    appInfo.title || _.get(appInfo, ['general', 'title'])
+  );
+
   // Not using escapeSpecialChars because we don't want to escape single quotes (not
   // allowed in JSON)
-  const description = legacyApp.general.description
+  const description = (
+    appInfo.description ||
+    _.get(appInfo, ['general', 'description']) ||
+    ''
+  )
     .replace(/\n/g, '\\n')
     .replace(/"/g, '\\"');
 
-  const runnerVersion = await getPackageLatestVersion(
-    'zapier-platform-legacy-scripting-runner'
-  );
+  const version = appDefinition.version
+    ? semver.inc(appDefinition.version, 'patch')
+    : '1.0.0';
 
-  const templateContext = {
-    name: _.kebabCase(legacyApp.general.title),
-    description,
-    appId: legacyApp.general.app_id || 'null',
-    cliVersion: PACKAGE_VERSION,
-    coreVersion: appDefinition.platformVersion,
-    runnerVersion
+  const dependencies = {
+    [PLATFORM_PACKAGE]: appDefinition.platformVersion
   };
+  if (appDefinition.legacy) {
+    const RUNNER_PACKAGE = 'zapier-platform-legacy-scripting-runner';
```

Comment:
this declaration should probably be moved to https://github.com/zapier/zapier-platform/blob/master/packages/cli/src/constants.js